easystage5.js

```diff
@@ -0,0 +1,7 @@
+<script>
+// When the user clicks on div, open the popup
+function revealhint() {
+  var popup = document.getElementById("myPopup");
```

Comment:
correct me if I am wrong, but where it the element with the ID of `myPopup` in your HTML code.